content/pt-br/_index.html

```diff
@@ -42,12 +42,12 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Assista Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona on Maio 20-23, 2019</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" id="desktopKCButton">>KubeCon na Europa de 16 a 20 de maio de 2022</a>
```

Comment:
There is an extra ">" in line number 45 after buttonid `id="desktopKCButton">>KubeCon na Europa`
Kindly update it as it can interfere with the HTML tags.

Reply:
Done. Thanks for pointing out.